wpilibNewCommands/src/main/native/include/frc2/command/Command.h

```diff
@@ -484,10 +484,11 @@ class Command : public wpi::Sendable, public wpi::SendableHelper<Command> {
  protected:
   Command();
 
+  std::optional<std::string> m_previousComposition;
```

Comment:
Why is this moving from private to protected? I don't think inheritors have reason to access it.

Reply:
This was a mistake when merging 🙃